### ISSUE TITLE
how to do the augmentation for training?

### ISSUE THREAD

**Issue by seyeeet** [yassouali/pytorch-segmentation#112]
Thanks for our code, I have a clarification question and appreciate your answer.
Let say I am augmenting the image, lets say I resize the image and then pad it and then crop a fix size.
for the padding areas, should I assign class 0 (background) or class 255 (which is ignore class)?
in other words, should I backpropagate the loss for the pad areas and make my network to consider them as 0?

Thanks a lot for your help.

**Comment by yassouali** [comment]
Hi @seyeeet 

In your case, if you pad, I think it is better to assign 255 / ignore label of your CE loss in the targets. There is no information to be learned in this case. For other photometric & geometric transformations, simply apply them to both the inputs & the targets.

Hope this helps.
